css/category.css

```diff
@@ -0,0 +1,26 @@
+@font-face { /* Bold font */
+    font-family: 'Futura bd';
```

Comment:
use `<link>` tag in order to preload the fonts before loading the whole page & Stylesheet
like this
```
 <link rel="preload" href="./assets/fonts/Fearlessly-Regular.woff2" as="font"
    type="font/woff2" >
```
    
  Also separate your fonts in another stylesheet ex. fonts.css

Reply:
done